src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.awt.*;
+
+public class Ball {
+    private String color;
+    private int number;
+
+
+
+
+    //geter seter col
```

Comment:
There are multiple empty lines in your code. It's a good practice to remove unnecessary empty lines to maintain a clean and readable code base. Please refer to the checklist item about redundant empty lines.